config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+StrongMigrations.start_after = 20190304222108
```

Comment:
To run for all migrations **after** our latest migration:
https://github.com/3scale/porta/blob/3f7fa1dc5b4904d1e9cbcb7b50436ea01c6c6322/db/schema.rb#L14